bundles/org.openhab.binding.tibber/README.md

```diff
@@ -0,0 +1,119 @@
+| Current Total      | Current Total Price (energy + tax)      | True      |
+| Starts At          | Current Price Timestamp                 | True      |
+| Daily Cost         | Daily Cost (last/previous day)          | True      |
+| Daily Consumption  | Daily Consumption (last/previous day)   | True      |
+| Daily From         | Timestamp (daily from)                  | True      |
+| Daily To           | Timestamp (daily to)                    | True      |
+| Hourly Cost        | Hourly Cost (last/previous hour)        | True      |
+| Hourly Consumption | Hourly Consumption (last/previous hour) | True      |
+| Hourly From        | Timestamp (hourly from)                 | True      |
+| Hourly To          | Timestamp (hourly to)                   | True      |
+
+Tibber Pulse:
+
+| Channel ID              | Description                              | Read-only |
+|-------------------------|------------------------------------------|-----------|
+| Timestamp               | Timestamp for live measurements          | True      |
+| Power                   | Live Power Consumption                   | True      |
+| Last Meter Consumption  | Last Recorded Meter Consumption          | True      |
+| Accumulated Consumption | Accumulated Consumption since Midnight   | True      |
+| Accumulated Cost        | Accumulated Cost since Midnight          | True      |
+| Currency                | Currency of Cost                         | True      |
+| Min Power               | Min Power Consumption since Midnight     | True      |
+| Average Power           | Average Power Consumption since Midnight | True      |
+| Max Power               | Max Power Consumption since Midnight     | True      |
+| Voltage 1-3             | Voltage per Phase                        | True      |
+| Current 1-3             | Current per Phase                        | True      |
+| Power Production        | Live Power Production                    | True      |
+| Accumulated Production  | Accumulated Production since Midnight    | True      |
+| Min Power Production    | Min Power Production since Midnight      | True      |
+| Max Power Production    | Max Power Production since Midnight      | True      |
+
+
+## Binding Configuration
+
+To access and initiate the Tibber binding, a Tibber account is required.
+
+Required input needed for initialization:
+
+* Tibber token
+* Tibber HomeId
+* Refresh Interval (min 1 minute)
+
+Note: Tibber token is retrieved from your Tibber account:
+[Tibber Account](https://developer.tibber.com/settings/accesstoken)
+
+Note: Tibber HomeId is retrieved from [www.developer.com](https://developer.tibber.com/explorer). Sign in (Tibber account) and load personal token. HomeId is retrieved by copying and running query below from Tibber API explorer. If Pulse is connected, realTimeConsumptionEnabled will report "true" for the associated HomeId. Copy desired HomeId, without quotation marks, and paste into PaperUI configuration.
```

Comment:
Can you improve the readability of these notes?
Also start sentences on new lines.

PaperUI -> Paper UI (fix all occurrences please)

Reply:
Corrected, and hopefully also managed to present the steps in a more readable manner.